index.html

```diff
@@ -0,0 +1,666 @@
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+        </section>
+    </div>
+    </main>
+
+    <footer class="footer1">
+        <div class="social-icons">
+        <i class="fab fa-facebook-square"></i>
+        <i class="fab fa-twitter"></i>
+        <i class="fab fa-youtube-square"></i>
+        <i class="fab fa-instagram-square"></i>
+        <i class="fab fa-pinterest-square"></i>
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-envelope"></i>
+         </div>
+        <ul class="thin">
+            <li>EVENTS</li>
```

Comment:
- [ ] The footer content has too much width

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88701963-119de180-d0e1-11ea-9881-e0989b7cf1c4.png)|![image](https://user-images.githubusercontent.com/61479648/88701986-195d8600-d0e1-11ea-9eab-182a98c7b50d.png) |

- [ ] The alignment of the bottom section is off. Make it more like the original.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88702115-4f026f00-d0e1-11ea-8741-0dcf372266e0.png) |
![image](https://user-images.githubusercontent.com/61479648/88702102-4ad65180-d0e1-11ea-8138-2945007d1686.png) |